aggregators/hop-aggregator/index.ts

```diff
@@ -0,0 +1,30 @@
+import { CHAIN } from "../../helpers/chains";
+import { httpPost } from "../../utils/fetchURL";
+import { FetchOptions } from "../../adapters/types";
+
+const fetchVolume = async (options: FetchOptions) => {
+  let timestamp = options.startOfDay * 1_000;
+
+  const res = await httpPost("http://35.153.229.202/api/volume", { timestamp });
+  const record = res.record || {};
+
+  return {
+    dailyVolume: record.total_volume_usd ? record.total_volume_usd.toFixed(2) : 0,
+    totalVolume: res.total_volume_usd ? res.total_volume_usd.toFixed(2) : 0,
+    dailyFees: 0,
```

Comment:
could you get daily value for fees and volume ?

Reply:
Yes, this is already working!

We charge 0 fees and will not change this.

As for the dailyVolume, in the response from the server we return it in the format ```{ total_volume_usd: 100, record: { total_voume_usd: 100 }}```. The record itself is for the day. Meanwhile, the first total_volume_usd is for overall.